libraries/AP_Math/quaternion.h

```diff
@@ -70,7 +70,7 @@ class QuaternionT {
     void        rotate(enum Rotation rotation);
 
     // convert a vector from earth to body frame
```

Comment:
Shouldn't this comment say "convert a vector from body to earth frame"?

Reply:
Haha, yes! Of all the comments I messed with I forgot to change the very comment that started this all